rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2190,8 +2191,9 @@ List<WorkflowDurationData> getTopPendingTimeWorkflows(@HeaderParam("sessionid")
     CompletedJobsCountData getCompletedJobs(@HeaderParam("sessionid") String sessionId,
             @QueryParam("myjobs") @DefaultValue("false") boolean myJobs,
             @QueryParam("workflowName") @DefaultValue("null") String workflowName,
-            @QueryParam("timeWindow") String timeWindow, @QueryParam("zoneId") @DefaultValue("null") String zoneId)
-            throws RestException;
+            @QueryParam("startdate") @DefaultValue("0") long startDate,
+            @QueryParam("enddate") @DefaultValue("0") long endDate,
+            @QueryParam("numberOfIntervals") int numberOfIntervals) throws RestException;
```

Comment:
I would prefer if endDate default value -1 with a meaning of now() (this should be implemented in the code).
numberOfIntervals default value should be 1